internal/coreinternal/scraperinttest/README.md

```diff
@@ -0,0 +1,5 @@
+# scraperinttest
+
+This package is intended to be a temporary workspace for extracting common functionality
+from scraper integration tests. Once stabilized, this code can likely move to `pdatatest`
+or its sub-packages `pmetrictest`, etc.
```

Comment:
The `pdatatest` is also experimental and internal :) Why don't we put it right there instead of creating another module?

Reply:
Isn't `pdatatest` under `pkg`?

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/pdatatest

Reply:
Oh, you're right. My bad. Ok. let's keep it here for now